### PR TITLE
[LTD-5121] Rename superseded by amendment status

### PR DESCRIPTION
**Pull request by currycoder**
### Aim

This change renames the status "Superseded by amendment" to "Superseded by exporter edit" as per the Jira.  This is for the purpose of better explaining things to users.

[LTD-5121](https://uktrade.atlassian.net/browse/LTD-5121)

[LTD-5121]: https://uktrade.atlassian.net/browse/LTD-5121?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/applications/views/tests/test_amendments.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def test_create_amendment(self):
         self.assertEqual(amendment_application.name, self.application.name)
         self.assertEqual(amendment_application.amendment_of_id, self.application.id)
         self.application.refresh_from_db()
-        self.assertEqual(self.application.status.status, "superseded_by_amendment")
+        self.assertEqual(self.application.status.status, "superseded_by_exporter_edit")
```
Comment (saruniitr) [inline api/applications/views/tests/test_amendments.py:34]: Could this also be updated to use enum?

File: api/staticdata/statuses/migrations/0014_rename_superseded_by_amendment_status.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from django.db import migrations
+
+
+def rename_case_status_superseded_by_amendment(apps, schema_editor):
+    CaseStatus = apps.get_model("statuses", "CaseStatus")
+
+    STATUS__SUPERSEDED_BY_AMENDMENT = "00000000-0000-0000-0000-000000000034"
+
+    status = CaseStatus.objects.get(
+        id=STATUS__SUPERSEDED_BY_AMENDMENT,
```
Comment (saruniitr) [inline api/staticdata/statuses/migrations/0014_rename_superseded_by_amendment_status.py:10]: Could these also be updated to match enum i.e., to use SUPERSEDED_BY_EXPORTER_EDIT.

**Review by saruniitr — APPROVED** [review]
with it updated to use enums it looks good to me.

**Review by depsiatwal — COMMENTED** [review]
been through this appears just to be a rename to make the action clearer so lgtm.

File: api/applications/views/tests/test_amendments.py (hunk 1 above)
Comment (currycoder) [reply api/applications/views/tests/test_amendments.py]: Done

File: api/staticdata/statuses/migrations/0014_rename_superseded_by_amendment_status.py (hunk 2 above)
Comment (currycoder) [reply api/staticdata/statuses/migrations/0014_rename_superseded_by_amendment_status.py:10]: To me, this sort of makes more sense as it is now.  "get the superseded by amendment status and change the name of it to "superseded by exporter edit".  What do you think?

Comment (saruniitr) [reply api/staticdata/statuses/migrations/0014_rename_superseded_by_amendment_status.py:10]: I think I added this comment at the wrong line, was suggesting to change line12 https://github.com/uktrade/lite-api/pull/2058/files#diff-34185a8fca9317ea15cf69dd6debac08dd0e9229b09da62add6c0de51aed271dR12 also to use enum

**Review by hnryjmes — APPROVED** [review]
looks good as is, one minor comment for style

**Pull request merged**